spec/create_table_statement_spec.cr

```diff
@@ -24,8 +24,6 @@ describe LuckyMigrator::CreateTableStatement do
       amount_paid decimal NOT NULL,
       email text);
     SQL
-
-    built.table_statement.should eq built.statements.first
```

Comment:
I made just one `statements` method instead of having separate ones that get built up

Reply:
Good call, I guess you don't need to access the index_statements separately, and this simplifies the class a low